garnbarn_api/tests/test_view.py

```diff
@@ -217,6 +226,101 @@ def test_delete(self):
         assignments_in_database = Assignment.objects.all()
         self.assertEqual(len(assignments_in_database), 0)
 
+    def test_get_tag(self):
+        """Return detail of the tag."""
+        response = self.client.get("/api/v1/tag/2/")
+        converted_data = convert_to_json(response.content)
+        expected_result = json.dumps({
+            "id": 2,
+            "name": "test_tag2",
+            "author": {
+                'line': None,
+                'name': 'user_name',
+                'uid': 'user_id'
+                },
```

Comment:
Why author returns the entire data for the user?

Reply:
Remove done in commit d34d7a6.